.travis.yml

```diff
@@ -22,7 +19,7 @@ addons:
     packages:
       - g++-4.8
 before_install:
-  - npm i -g npm@^2.0.0
+  - npm i -g npm@^3.0.0
```

Comment:
just recording that we're going to upgrade to NPM 3 in production to follow this